[MacOS only] openroad cmake-build error: undeclared identifier 'lefyyparse'

On an M1 Mac running Ventura and xcode command line utilities and Python v3.10.9  
Following the clone + build instructions in the docs at
https://docs.siliconcompiler.com/en/latest/user_guide/installation.html#siliconcompiler

When running `python -m pip install -e .` the build failed with  

> zeroasic/sc/siliconcompiler/_skbuild/macosx-13.0-arm64-3.10/cmake-build/_deps/openroad-src/src/odb/src/lef/lef/lefrReader.cpp:297:12: error: use of undeclared identifier 'lefyyparse'

A google search led to https://github.com/The-OpenROAD-Project/OpenROAD/issues/1688 (@nmoroze). I followed the suggested advice in the [comment](https://github.com/The-OpenROAD-Project/OpenROAD/issues/1688#issuecomment-1087744398), and brew installed bison and flex (with override of the existing /usr/bin flavors).

This resolved the error 😄 

Perhaps adding a troubleshooting section to the docs or README would help others who run into this.

--- a/docs/user_guide/installation.rst
+++ b/docs/user_guide/installation.rst
@@ -92,6 +92,19 @@ To exit the Python virtual environment, type 'deactivate' and hit enter.
 You can also install SiliconCompiler from the latest `SiliconCompiler GitHub Repository <https://github.com/siliconcompiler/siliconcompiler>`_. This option is currently
 only supported on Linux/MacOS platforms.
 
+On MacOS, note that you must first install Bison and Flex from Homebrew.
+
+.. code-block:: bash
+
+   brew install bison
+   brew install flex
+
+Ensure that the path to the Homebrew packages takes priority over system
+packages in your ``$PATH``. Run ``brew --prefix`` to determine where Homebrew
+installs packages on your machine.
+
+Finally, to clone and install SiliconCompiler, run the following.
+
 .. code-block:: bash
 
    git clone https://github.com/siliconcompiler/siliconcompiler

